ivy/src/main/scala/sbt/internal/librarymanagement/MakePom.scala

```diff
@@ -259,7 +260,7 @@ class MakePom(val log: Logger) {
         types match {
           case Nil                                     => Artifact.PomType
           case xs if xs.contains(Artifact.DefaultType) => Artifact.DefaultType
-          case x :: xs                                 => x
+          case x :: (xs @ _)                           => x
```

Comment:
Avoiding the compiler warning. Could also be `_`, but I grew up writing `x :: xs`.